courier/courier_test.go

```diff
@@ -162,3 +170,67 @@ func TestSMTP(t *testing.T) {
 	assert.Contains(t, string(body), `"test-stub-header1":["foo"]`)
 	assert.Contains(t, string(body), `"test-stub-header2":["bar"]`)
 }
+
+func TestQueueSMS(t *testing.T) {
+	if testing.Short() {
+		t.SkipNow()
+	}
```

Comment:
Since we do not use any docker stuff or anything else, I think we can execute this with short tests as well!

```suggestion
```

Reply:
Good point. Removed